website/docs/workloads/fio/fio-profiles.md

```diff
@@ -269,6 +269,20 @@ This profile uses an algorithm to determine the total number of jobs/threads as
   ./VirtualClient --profile=PERF-IO-FIO-DISCOVERY.json --system=Demo --timeout=1440 --packageStore="{BlobConnectionString|SAS Uri}"  --parameters="QueueDepths="4,16,256"
   ```
 
+-----------------------------------------------------------------------
+
+### PERF-IO-FIO-MULTITHROUGHPUT.json
+
```

Comment:
There is already a PERF-IO-FIO-MULTITHROUGHPUT.json section below, could you merge the two? And also please give more concrete examples on the commands running on the random io disks and sequential io disks, concurrently or not concurrently